applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -879,23 +1012,46 @@ namespace Kratos
                 noalias( rValues[i] ) = mConstitutiveLawVector[i]->GetValue( STRESSES, rValues[i] );
```

Comment:
yes i see. However deos anyone in @KratosMultiphysics/structural-mechanics have a hint about that?